app/controllers/attachments_controller.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+class AttachmentsController < SecureController
+  def create
+    Attachment.create!(params.permit(permitted_params))
+    head :created
+  end
+
+  def update
+    load_attachment.update(params.permit(permitted_params))
```

Comment:
If attachment isn't found, this will raise an exception, method update not found. Will this generate the correct jsonapi response?